packages/gasket-cli/src/scaffold/actions/prompt-hooks.js

```diff
@@ -61,7 +61,7 @@ async function execPluginPrompts(context, plugins = [], presets = []) {
   //
   // @see: https://github.com/SBoudrias/Inquirer.js/#inquirercreatepromptmodule---prompt-function
   //
-  const prompt = inquirer.createPromptModule();
+  const prompt = context.prompts ? inquirer.createPromptModule() : () => {};
```

Comment:
Does this return an empty object, or do you need to wrap it with parentheses?
```suggestion
  const prompt = context.prompts ? inquirer.createPromptModule() : () => ({});
```

Add a test for this if you can.